debian/control

```diff
@@ -20,5 +20,8 @@ Depends: php-mysql,
          php (>= 7~),
          libapache2-mod-php,
          ${misc:Depends}
-Description: web-accessible database for longitudinal multi-site studies
- Whis should I install this one over XNAT?
+Description: LORIS is a web-based data management software. 
+ It is an OPEN SOURCE framework for storing and processing behavioural, 
```

Comment:
Everything in Debian proper is open source.  Also start as it is a new paragraph where "It" is not yet defined, so smth like "LORIS is a framework for ..."